nsxt/resource_nsxt_policy_mac_discovery_profile.go

```diff
@@ -98,28 +163,20 @@ func resourceNsxtPolicyMacDiscoveryProfileCreate(d *schema.ResourceData, m inter
 		return err
 	}
 
+	// TODO - consider including standard object attributes in the schema
+	tags := getPolicyTagsFromSchema(d)
```

Comment:
Do we want to include `tags`, `display_name` and `description` in the extended schema?

Reply:
It's not hard to improve `GetExtendedSchema` and make it return common extended schema for some well-known policy attrs, but it'll need some refactor on the test side of stuff to make everything e2e working. Can we tackle this after the polymorphic struct is addressed?